### PR TITLE
tripStore trpc conversion 

### PR DESCRIPTION
**Pull request by alexander01202**

### CODE REVIEW

File: client/hooks/feed/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { usePublicFeed } from './publicFeed';
 import { useUserPacks } from './../packs';
-import { useUserTrips } from './../trips';
```
Comment (andrew-bierman) [inline client/hooks/feed/index.js:3]: Why this change ?

File: client/components/EditableText/index.js [hunk 2]
```diff
@@ -57,8 +58,8 @@ export const EditableInput = ({
             setEditTitle(false);
             titleRef.current.style =
               'font-size:20px !important;font-weight:bold;color: #22c67c;';
-
-            dispatch(editTrip(tripDetails));
+            useEditTrips(tripDetails)
```
Comment (andrew-bierman) [inline client/components/EditableText/index.js:61]: pretty sure we should be destructuring the mutation func out of here and calling it ? or am I mistaken

File: client/hooks/feed/index.js (hunk 1 above)
Comment (alexander01202) [reply client/hooks/feed/index.js:3]: The trpc query hook was for singleTripsStore but was saved in a folder called trips instead of singleTrips, which was stopping me from saving my own trpc hook (which was the actually trpc trips hook) in a trips folder. I don't know if my explanation makes sense to you?

File: client/components/EditableText/index.js (hunk 2 above)
Comment (alexander01202) [reply client/components/EditableText/index.js]: My bad. I'd fix that right away

**Comment by alexander01202** [comment]
Fixed now andrew.

File: client/hooks/feed/index.js (hunk 1 above)
Comment (andrew-bierman) [reply client/hooks/feed/index.js:3]: i see what you are saying, but user trips will return more than 1 trip, so i think we should name it something else ? or maybe i'm misunderstanding

Comment (alexander01202) [reply client/hooks/feed/index.js:3]: No it wouldn't. All I did was rename the folder, so I can use the previous folder name for mine instead. There are no double files or double calls of `useUserTrips`. It's still the exact same folder/file structure, just with a different folder name. (from trips -> singletrips)

Comment (andrew-bierman) [reply client/hooks/feed/index.js:3]: Got it thanks for clarifying
